openedx/core/djangoapps/util/user_messages.py

```diff
@@ -143,7 +143,7 @@ def register_warning_message(self, request, message, **kwargs):
         """
         Registers a warning message to be shown to the user.
         """
-        self.register_user_message(request, UserMessageType.WARNING, message, **kwargs)
+        self.register_user_message(request, UserMessageType.WARNING, message, once_only=True, **kwargs)
```

Comment:
This kept happening on redirect (see image), and adding `once_only=True` looks like a simple fix 🤷 
<img width="400" alt="Outline tab rendering duplicate logistration warnings" src="https://user-images.githubusercontent.com/25124041/100651875-22bb6100-3314-11eb-838c-40b213871269.png">

Reply:
Right idea, wrong execution